Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM archlinux:latest
+LABEL contributor="shadowapex@gmail.com"
+
+RUN echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist\n" >> /etc/pacman.conf && \
+	pacman --noconfirm -Syy && \
+	pacman --noconfirm -Syu && \
+	pacman --noconfirm -S arch-install-scripts btrfs-progs pyalpm sudo && \
+	pacman --noconfirm -S --needed base-devel git && \
+	echo "%wheel ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
```

Comment:
Do we even need sudo? The build runs as root

Reply:
Yes, `sudo` is needed as pikaur runs as a non-root user. This is done because pikaur spawns package builds using `systemd-run` to build packages as non-root, but the container does not have systemd running as its init process.